.gitignore

```diff
@@ -2,3 +2,4 @@
 dist
 node_modules
 oclif.manifest.json
+.earthfast/rpc_api_key
```

Comment:
this should go outside the repository in your computer home dir like ~/.earthfast/rpc_api_key. there's already a ~/.earthfast dir that we have